workers/data_refinery_workers/processors/smashing_utils.py

```diff
@@ -1033,7 +1033,7 @@ def download_computed_file(download_tuple: Tuple[ComputedFile, str]):
         logger.exception("Failed to sync computed file", computed_file_id=latest_computed_file.pk)
 
 
-def sync_quant_files(output_path, samples: List[Sample]):
+def sync_quant_files(output_path, samples: List[Sample], filtered_samples: Dict):
```

Comment:
Could you note in the docstring that this function modifies `filtered_samples` to include samples that it's unable to sync?